lib/Arch/Sleigh/ControlFlowStructuring.cpp

```diff
@@ -127,6 +135,16 @@ CoarseCategoryFromFlows(const std::map<size_t, CoarseFlow> &ops) {
     return CoarseCategory::kCatNormal;
   }
 
+  auto all_normal_or_intra_effects = std::all_of(
+      ops.begin(), ops.end(), [](const std::pair<size_t, CoarseFlow> &op) {
```

Comment:
Extract lambda as local var to improve readability.

Reply:
[7245171](https://github.com/lifting-bits/remill/pull/645/commits/7245171900f55e1665acbcbed385aa6b02875c06)